main.js

```diff
@@ -1,36 +1,32 @@
-//variables
-let user = 'John Doe';
-console.log(user);
-let student = 'Sergey';
-console.log(student);
-user = student; // Sergey
-console.log(user);
+//Функция palindrome
```

Comment:
You keep your code in script.js and main.js
Please, think about how you will save your code, this file structure looks messy. Tidy up =)